orttraining/orttraining/python/training/ortmodule/_custom_op_symbolic_registry.py

```diff
@@ -663,3 +663,33 @@ def einsum_internal(g, equation, tensor_list):
 
 
 # End of torch.einsum.
+
+
+@register_symbolic("group_norm")
+def group_norm(g, input, num_groups, weight, bias, eps, cudnn_enabled):
+    # Torch's group_norm's weight and bias are optional, its gradient has a bool[3] augment to indicate
+    # whether to compute the gradient for input, weight, bias. For simplicity of the gradient graph builder,
+    # we support only the case that weight and bias are not None.
+    from torch.onnx.symbolic_opset9 import group_norm as group_norm_generic
+
+    if weight is None or sym_help._is_none(weight) or bias is None or sym_help._is_none(bias):
+        return group_norm_generic(g, input, num_groups, weight, bias, eps, cudnn_enabled)
+
+    shape = g.op("Shape", input)
```

Comment:
Can you share the link to the pytorch code that does this in going from `F.group_norm` to `native_group_norm`?

Reply:
https://github.com/pytorch/pytorch/blob/77d29bcee200f04bece4a86283acfb8e1ec830ad/aten/src/ATen/native/group_norm.cpp#L152